docs/advanced/custom-renderer.md

```diff
@@ -55,23 +55,23 @@ This is a live preview where you can see the changes when you:
 
 ## Developing the Document Renderer
 
-Now that we have set up the development environment, we can start writing our document renderer. We will first define the data structure of our 📜 Certificate of Completion, followed by writing the renderer to render the HTML code corresponding to the data provided.
+Now that we have set up the development environment, we can start writing our document renderer. We will first define the data structure of our 📜 Certificate of Completion (COC), followed by writing the renderer to render the HTML code corresponding to the data provided.
 
 ### Update sample document data and type
 
 To update the raw document data and the corresponding data type, you will need to update the data definition file in `src/templates/sample.ts`:
 
-```js
+```typescript jsx
 import { Document } from "@govtechsg/decentralized-renderer-react-components";
 
-export interface CustomTemplateCertificate extends Document {
+export interface CocTemplateCertificate extends Document {
```

Comment:
I followed through the step and ran into an error, which was traced back to this.

Reason being: Integration tests make use of this, specifically this line `import { customTemplateCertificate } from "../templates/sample";` is causing the build error.

We could add an instruction to delete the integration tests as well.

Reply:
Specifically, this is the file: https://github.com/gjj/test-new-demo/blob/ebbcb3ae63183436717435cecf6aa81553f6aaa7/src/integration/integration.spec.ts

The error on Netlify (and locally is the same too): https://app.netlify.com/sites/nifty-noether-8b2f72/deploys/5f27f96d0b21193aea00ad6a

Reply:
error at which step in the doc ? build ?

Reply:
yep, either (a) `npm run build` locally or (b) on Netlify, and that's cos of the integration tests.

Reply:
Added a step to clean the repo first